src/main/java/com/google/devtools/build/lib/rules/filegroup/Filegroup.java

```diff
@@ -94,7 +94,7 @@ public ConfiguredTarget create(RuleContext ruleContext)
             new Runfiles.Builder(
                     ruleContext.getWorkspaceName(), configuration.legacyExternalRunfiles())
                 .addTransitiveArtifacts(filesToBuild)
-                .addDataDeps(ruleContext)
+                .addRunfiles(ruleContext, RunfilesProvider.DATA_RUNFILES)
```

Comment:
@lberki If you want this to be `DEFAULT_RUNFILES` so that it's not changed again later, I'm okay with that too.

Reply:
Meh let's not poke that beast